src/main/scala/uk/gov/hmrc/domain/Nino.scala

```diff
@@ -33,8 +33,8 @@ object Nino {
   implicit val ninoWrite: Writes[Nino] = new SimpleObjectWrites[Nino](_.value)
   implicit val ninoRead: Reads[Nino] = new SimpleObjectReads[Nino]("nino", Nino.apply)
 
-  private val validNinoFormat = "[[A-Z]&&[^DFIQUV]][[A-Z]&&[^DFIQUVO]] ?\\d{2} ?\\d{2} ?\\d{2} ?[A-Z]{1}"
-  private val invalidPrefixes = List("BG", "GB", "NK", "KN", "TN", "NT", "ZZ")
+  private val validNinoFormat = "[[A-Z]&&[^DFIQUV]][[A-Z]&&[^DFIQUVO]] ?\\d{2} ?\\d{2} ?\\d{2} ?[A-D]{1}"
+  val invalidPrefixes = List("BG", "GB", "NK", "KN", "TN", "NT", "ZZ")
```

Comment:
I don't think this should be publicly available outside of this library as they are implementation details. Can we make it package protected or something similar?